### DISCUSSION TITLE
argo-workflows/node-field-selector/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#10479] [General]
# argo-workflows/node-field-selector/

https://argoproj.github.io/argo-workflows/node-field-selector/

**Comment by lordsoffallen** [comment]
We are using DAGs and we have the cases where we can't refer to name field. Our name value looks like this:         

`workflow-0.2.0-smfdx.parallel-etl-train-pipeline(0:countries:["ae","at","b2"],regionLong:eu-west-1,regionShort:eu).parallel-etl-train-pipeline(0:ae).validate-data.pod(0)`

It complains about the quotes. Could we maybe provide an example where this would work or support using the id field? That one looks like it would work but currently not supported afaik.
